src/selectors/nodes.js

```diff
@@ -124,6 +127,7 @@ export const getNodeData = createSelector(
         id,
         name: nodeName[id],
         type: nodeType[id],
+        icon: nodeDatasetType[id] || nodeType[id],
```

Comment:
In referencing that the plot data is saved under `node.plot`, we should use that field to reference whether the icon is a plot icon rather than using `datasetType`?

Reply:
@studioswong  -- as per Lim's new back-end. this works.